kafl_fuzzer/config_default.yaml

```diff
@@ -0,0 +1,28 @@
+# kAFL configuration
```

Comment:
Since most of the content of this file doesn't change, it would be better to make a `git mv`

Reply:
I think it was moved and then modified on same commit...the comments are changed to reflect changed semantics